R/test-files.R

```diff
@@ -59,21 +59,25 @@ test_files <- function(paths, reporter = "summary",
   invisible(testthat_results(results))
 }
 
-
-#' Find the test files.
-#' @param path path to tests
+#' Filter File List for Tests
+#'
+#' Filtering is done by \code{grep}ing with pattern \code{filter} against
+#' the basename from \code{files} with any leading \sQuote{test-?} and trailing
+#' \sQuote{.[Rr]} removed.
+#'
+#' @param files character vector of file names to filter
 #' @param filter cf \code{\link{test_dir}}
 #' @param invert If \sQuote{TRUE} return files which do \emph{not} match.
 #' @param ... Additional arguments passed to \code{grepl} to control filtering.
 #' @return the test file paths
 #' @keywords internal
 #' @export
-find_test_scripts <- function(path, filter = NULL, invert = FALSE, ...) {
-  files <- dir(path, "^test.*\\.[rR]$", full.names = TRUE)
+
+filter_test_scripts <- function(files, filter = NULL, invert = FALSE, ...) {
```

Comment:
I'm just following the pattern for `find_test_scripts` of documenting with `@keywords internal` and exporting.  Do you want me to remove docs for both the original and the new function, or just for the new function.